package mpsqual;

message EthernetRequest { 
	enum RequestTypeT {
		STOP = 0;
		RUN	= 1;
		REPORT = 2;
	}
	
	required RequestTypeT requestType = 1 [default = STOP];
	required string local = 2 [default = ""];
	optional string remote = 3 [default = ""];
}

message EthernetResponse {
	enum AppStateT {
		STOPPED	= 0;
		RUNNING	= 1;
	}
		
	required AppStateT state = 1;
	required string local = 2;
	required string result = 3;
}
